utils/fixed_decimal/src/decimal.rs

```diff
@@ -2188,165 +1984,139 @@ fn test_float() {
         // HalfExpand tests
         TestCase {
             input: 1.234567,
-            precision: DoublePrecision::Magnitude(-2, RoundingMode::HalfExpand),
+            precision: DoublePrecision::Magnitude(-2),
             expected: "1.23",
         },
         TestCase {
             input: 1.235567,
-            precision: DoublePrecision::Magnitude(-2, RoundingMode::HalfExpand),
+            precision: DoublePrecision::Magnitude(-2),
             expected: "1.24",
         },
         TestCase {
             input: 1.2002,
-            precision: DoublePrecision::Magnitude(-3, RoundingMode::HalfExpand),
+            precision: DoublePrecision::Magnitude(-3),
             expected: "1.200",
         },
         TestCase {
             input: 888999.,
-            precision: DoublePrecision::Magnitude(2, RoundingMode::HalfExpand),
+            precision: DoublePrecision::Magnitude(2),
             expected: "889000",
         },
         TestCase {
             input: 888999.,
-            precision: DoublePrecision::Magnitude(4, RoundingMode::HalfExpand),
+            precision: DoublePrecision::Magnitude(4),
             expected: "890000",
         },
         TestCase {
             input: 0.9,
-            precision: DoublePrecision::Magnitude(0, RoundingMode::HalfExpand),
+            precision: DoublePrecision::Magnitude(0),
             expected: "1",
         },
         TestCase {
             input: 0.9,
-            precision: DoublePrecision::Magnitude(2, RoundingMode::HalfExpand),
-            expected: "0",
+            precision: DoublePrecision::Magnitude(2),
+            expected: "00",
```

Comment:
the result of "0.9".half_even(2) is "00"

Reply:
@sffc , is that okay?

Reply:
yes